topics/sequence-analysis/tutorials/ref-based-rad-seq/tutorial.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: tutorial_hands_on
 topic_name: sequence-analysis
-tutorial_name: ref-based-rad-seq
+tutorial_name: denovo-rad-seq
```

Comment:
Mh, but this is inside the ref-based-rad-seq folder?

Reply:
Ooops You're right .... Too much red wine for the new year I think Sorry !